v2/internal/system/packagemanager/zypper.go

```diff
@@ -60,7 +62,9 @@ func (z *Zypper) PackageInstalled(pkg *Package) (bool, error) {
 	if pkg.SystemPackage == false {
 		return false, nil
 	}
-	stdout, _, err := shell.RunCommand(".", "zypper", "info", pkg.Name)
+	var env []string
+	env = shell.SetEnv(env, "LANGUAGE", "en_US.utf-8")
```

Comment:
Do we know this isn't going to break non US English installations?

Reply:
Yes, my OS was Chinese, if we don't set `LANGUAGE` env variable here, the `zypper info pkgName` command will return result in Chinese, so `wails doctor` will nott work properly here.